students/gene/project/node_db.py

```diff
@@ -0,0 +1,131 @@
+
+import sqlite3
+from sqlite3 import Error
+
+
+class Node:
+
+    def __init__(self, node_id, site_id='', management_ip='', console_id='', vendor_id='', platform_id='', os_id=''):
+        self.node_id = node_id
+        self.site_id = site_id
+        self.management_ip = management_ip
+        self.console_id = console_id
+        self.vendor_id = vendor_id
+        self.platform_id = platform_id
+        self.os_id = os_id
+
+
+    @property
+    def description(self):
+        return ("""Node: {}

```

Comment:
the built in textwrap.dedent() function is handy for this sort or thing:

https://docs.python.org/3/library/textwrap.html?highlight=dedent#textwrap.dedent